ios/KMMIC/Sources/Presentation/Modules/SurveyQuestion/Views/EmojiAnswerView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  EmojiAnswerView.swift
+//  KMMIC
+//
+//  Created by MarkG on 08/05/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct EmojiAnswerView: View {
+
+    let emojis: [String]
+    var highlightStyle: EmojiHighlightStyle = .leftItems
+
+    @Binding var selected: Int
```

Comment:
I prefer the name `selectedIndex` it will be more clear

Reply:
@phongvhd93 updated at [842c412](https://github.com/markgravity/kmm-ic/pull/89/commits/842c41203ebe9cceeb7e2fb098daaa37a1017378)